commands/main_test.go

```diff
@@ -0,0 +1,260 @@
+package commands_test
+
+import (
+	"database/sql"
+	"encoding/base64"
+	"fmt"
+	"io"
+	"log"
+	"os"
+	"strconv"
+	"strings"
+	"testing"
+
+	"github.com/bwmarrin/discordgo"
+	_ "github.com/mattn/go-sqlite3"
+)
+
+const (
+	testChannelID      = "123456789"
+	testJoke1          = "A steak pun is a rare medium well done."
+	testJoke2          = "What do you call a fish with no eyes?"
+	testJoke2Punchline = "Fsh"
+	testFileName1      = "file1.png"
+	testFileName2      = "file2.png"
+
+	testFile1 = "iVBORw0KGgoAAAANSUhEUgAAAAIAAAACCAIAAAD91JpzAAABhGlDQ1BJQ0MgcHJvZmlsZQAAKJF9kT1Iw1AUhU/TSkUqDnYQcchQ" +
+		"O1kQFXHUKhShQqgVWnUweekfNGlIUlwcBdeCgz+LVQcXZ10dXAVB8AfE0clJ0UVKvC8ptIjxwuN9nHfP4b37AKFZZZoVGgc03TYzqaSYy6+K" +
+		"4VeEMIQAYojLzDLmJCkN3/q6p06quwTP8u/7s/rVgsWAgEg8ywzTJt4gnt60Dc77xFFWllXic+Ixky5I/Mh1xeM3ziWXBZ4ZNbOZeeIosVjq" +
+		"YqWLWdnUiKeIY6qmU76Q81jlvMVZq9ZZ+578hZGCvrLMdVojSGERS5AgQkEdFVRhI0G7ToqFDJ0nffzDrl8il0KuChg5FlCDBtn1g//B79la" +
+		"xckJLymSBHpeHOdjFAjvAq2G43wfO07rBAg+A1d6x19rAjOfpDc6WuwIGNgGLq47mrIHXO4AQ0+GbMquFKQlFIvA+xl9Ux4YvAX61ry5tc9x" +
+		"+gBkaVbpG+DgEIiXKHvd59293XP7t6c9vx9wdHKmBJh2ogAAAAlwSFlzAAAuIwAALiMBeKU/dgAAAAd0SU1FB+YEGAEnEd1Tcy8AAAAZdEVY" +
+		"dENvbW1lbnQAQ3JlYXRlZCB3aXRoIEdJTVBXgQ4XAAAAFUlEQVQI1wXBAQEAAACAEP9PF1CpMCnkBftjnTYAAAAAAElFTkSuQmCC"
+	testFile2 = "iVBORw0KGgoAAAANSUhEUgAAAAIAAAACCAIAAAD91JpzAAABhGlDQ1BJQ0MgcHJvZmlsZQAAKJF9kT1Iw1AUhU/TSkUqDnYQcchQ" +
+		"O1kQFXHUKhShQqgVWnUweekfNGlIUlwcBdeCgz+LVQcXZ10dXAVB8AfE0clJ0UVKvC8ptIjxwuN9nHfP4b37AKFZZZoVGgc03TYzqaSYy6+K" +
+		"4VeEMIQAYojLzDLmJCkN3/q6p06quwTP8u/7s/rVgsWAgEg8ywzTJt4gnt60Dc77xFFWllXic+Ixky5I/Mh1xeM3ziWXBZ4ZNbOZeeIosVjq" +
+		"YqWLWdnUiKeIY6qmU76Q81jlvMVZq9ZZ+578hZGCvrLMdVojSGERS5AgQkEdFVRhI0G7ToqFDJ0nffzDrl8il0KuChg5FlCDBtn1g//B79la" +
+		"xckJLymSBHpeHOdjFAjvAq2G43wfO07rBAg+A1d6x19rAjOfpDc6WuwIGNgGLq47mrIHXO4AQ0+GbMquFKQlFIvA+xl9Ux4YvAX61ry5tc9x" +
+		"+gBkaVbpG+DgEIiXKHvd59293XP7t6c9vx9wdHKmBJh2ogAAAAlwSFlzAAAuIwAALiMBeKU/dgAAAAd0SU1FB+YEGAEnJmXu1iAAAAAZdEVY" +
+		"dENvbW1lbnQAQ3JlYXRlZCB3aXRoIEdJTVBXgQ4XAAAAEklEQVQI12P4//8/AwT8//8fACnkBft7DmIIAAAAAElFTkSuQmCC"
```

Comment:
IMO better idea to store these as static data for tests in a directory. as .pngs that is that the tests will load into memory to compare.

Reply:
Done.